_docs/data-workflow.md

```diff
@@ -15,16 +15,16 @@ One function of the Glider DAC is to coordinate with the NWS' National Data Buoy
 
 ### NDBC Glider Page
 
-Active and historical glider deployments are posted to an [NDBC glider page](http://www.ndbc.noaa.gov/gliders.php).  These questions and answers from NDBC clarify the information on the glider page specific information and GTS.  
+Active and historical glider deployments are posted to an [NDBC glider page](https://www.ndbc.noaa.gov/gliders.php).  These questions and answers from NDBC clarify the information on the glider page specific information and GTS.  
```

Comment:
https://www.ndbc.noaa.gov/gliders.php is a NDBC 404 page! I cannot find a replacement for this one.